src/templates/newengland-organizing.html

```diff
@@ -0,0 +1,102 @@
+                <p><a href="https://docs.google.com/spreadsheets/d/1-qjvGF3loVR0sr-AaDGZE2C7YgwcC1dP2wdPc3BT8fM/edit?usp=sharing">Here</a>’s a link to the competition organization spreadsheet. It will be discussed later on this page. Please go to this page, go to File &gt; Make a Copy, and rename it to your competition.</p>
+                <h2 id="are-you-qualified">Are you qualified?</h2>
+                <p>If you’re looking to organize, you should have a very good understanding of how competitions work. This doesn’t mean you need to know everything, but you should have helped out significantly during at least a few competitions. If you have helped out significantly, you’ve probably interacted with a Delegate a bit and they likely at least know who you are. In short, you should have a general idea of what events are popular, how long stuff takes, and what staff are needed to do what.</p>
+                <p>Even if you have helped out in the past and have a good idea of how everything works, it is much easier to organize with a friend. We would suggest that you do this for at least your first competition, just because of the time commitment required. We do have some local organizers who we may be able to pair you up with to help you get the hang of some of these steps.</p>
+                <h2 id="getting-in-touch">Getting in touch</h2>
+                <p>Once you are ready to contact us, please feel free to send us an email to <a href="mailto:newengland@cubingusa.org">newengland@cubingusa.org</a>. This address goes to all local Delegates and some veteran organizers who are happy to help out. We do our best to respond promptly, but if you don’t hear back for a few days, please send a follow-up email to make sure yours didn’t get lost.</p>
+                <h2 id="role-of-the-organizer-and-the-delegate">Role of the organizer and the Delegate</h2>
+                <p>The organizer and Delegate have several roles, but these can sometimes vary by region throughout the world. Here we want to describe exactly what the Delegate will do, and what the organizer (you) will need to do.</p>
+                <h3 id="choosing-the-date">Choosing the date</h3>
+                <p>Competitions have to be about 3 weeks apart if they’re within 100km of each other by WCA Regulations. Once you start with the organizing process, the Delegate will share a spreadsheet with you with upcoming competitions so we can ensure your competition doesn’t conflict with other competitions. Of course, the date must also work for the Delegate, organizer(s), and the venue itself. It’s best to choose a date that doesn’t conflict with a major holiday (like winter break) or other dates when students might be busy or away.</p>
```

Comment:
A thought...does this give the organizer too much of an impression that they get to pick the date?  Would it be better to replace this with something like "We'll work with you to pick a date that works for you, the Delegates' schedules, and the schedule of other competitions in the region"?  I don't love it when organizers email saying the day they want to hold a competition (and giving them more reason to believe they picked a good day by following the rules here, rather than talking to us, doesn't help).  I'd prefer the workflow of collaboratively picking the date while the delegate is responsible for ensuring it's a good day.